### ISSUE TITLE
grunt build not populating vendor.js

### ISSUE THREAD

**Issue by JaKXz** [angular-fullstack/generator-angular-fullstack#164]
I've been trying to deploy my app to heroku using the given steps, but unfortunately, my `dist/public/scripts/*.vendor.js` is completely empty (and as a result I get this error: `Uncaught ReferenceError: angular is not defined 5ee656d2.scripts.js:1`). I took the code block below from my `index.html` and I haven't modified my Gruntfile since generating the project. What am I missing?

_One thing to note: I did start the project with just the yeoman angular generator, and then merged in the angular-fullstack later on..._ my app works fine locally, but is that where the issue is coming from?

```
<!-- build:js scripts/vendor.js -->
<!-- bower:js -->
<script src="bower_components/jquery/dist/jquery.js"></script>
<script src="bower_components/angular/angular.js"></script>
<script src="bower_components/bootstrap/dist/js/bootstrap.js"></script>
<script src="bower_components/angular-resource/angular-resource.js"></script>
<script src="bower_components/angular-cookies/angular-cookies.js"></script>
<script src="bower_components/angular-sanitize/angular-sanitize.js"></script>
<script src="bower_components/angular-route/angular-route.js"></script>
<script src="bower_components/angular-bootstrap/ui-bootstrap-tpls.js"></script>
<!-- endbower -->
<!-- endbuild -->
```

What would be really cool is if components like angular could have a CDN url specified in production, with these files being the fallback.

**Comment by mcpDESIGNS** [comment]
I was also getting the "Uncaught ReferenceError: angular is not defined" after trying to set everything up, but doing a quick "grunt build" fixed it. Now everything is working!

My question is, I won't have to constantly enter this command in correct? (Let's say I'm adding some new files / etc)

**Comment by kaushik-evani** [comment]
I spent lots and lots of frustrated days because of this. I am using generator-angular-fullstack version 3.0.2. For me the css files in the routes were getting loaded fine
(i.e. `<!-- build:css({.tmp,client}) app/app.css -->`), but the vendor.css was empty. Bootstrap and font-awesome were not getting loaded. Original line which was buggy was 
`<!-- build:css({client}) app/vendor.css -->`
And for me it worked after I changed it to,
`<!-- build:css({.tmp,client}) app/vendor.css -->`
This small thing was not easy to find though. Found a lot of help in this thread. Cheers everyone! I can sleep in peace.
